### ISSUE TITLE
OOM

### ISSUE THREAD

**Issue by qiuqi-ovopark** [penfeizhou/APNG4Android#224]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | vivoS12  V2162A                      |      |
| System Version               | 10                                             |      |
| APNG4Android Library Version | 2.24.0                                        |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

java.lang.OutOfMemoryError: Failed to allocate a 72 byte allocation with 1627448 free bytes and 1589KB until OOM, target footprint 536870912, growth limit 536870912; giving up on allocation because <1% of heap free after GC.
	at android.os.Message.obtain(Message.java:168)
	at android.os.Handler.getPostMessage(Handler.java:999)
	at android.os.Handler.post(Handler.java:437)
	at com.github.penfeizhou.animation.FrameAnimationDrawable.onRender(FrameAnimationDrawable.java:235)
	at com.github.penfeizhou.animation.decode.FrameSeqDecoder$1.run(FrameSeqDecoder.java:64)
	at android.os.Handler.handleCallback(Handler.java:1013)
	at android.os.Handler.dispatchMessage(Handler.java:101)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:328)
	at android.os.HandlerThread.run(HandlerThread.java:67)

![Snipaste_2024-09-23_16-53-54](https://github.com/user-attachments/assets/2783a4b2-bf0e-4c72-862f-56117228e663)
![chat_gpt](https://github.com/user-attachments/assets/8ffea65d-c47c-413f-96f3-3bea78f818e8)
